docs/schemas-tests/XR-native-pa-yaml/Single-File-App.pa.yaml

```diff
@@ -0,0 +1,30 @@
+App:
+  Properties:
+    BackEnabled: =true
+    Theme: =PowerAppsTheme
+
+  Children:
+    Host:
+      Properties:
+        OnCancel: =false
+        OnEdit: =false
+        OnNew: =false
+        OnSave: =false
+        OnView: =false
+
+Screens:
+  Screen1:
```

Comment:
Should screens be an ordered map?

Reply:
per XR discussions, screens are not ordered.
The decision was made to keep the screen ordering in editor state.
The only thing important for the running of the app is the `App.StartScreen` rule.